dotcom-rendering/scripts/webpack/webpack.config.js

```diff
@@ -131,7 +131,9 @@ module.exports = [
 			sessionId,
 		}),
 	),
-	...(BUILD_VARIANT
+	// Only build the variant if in production mode
+	// Use `make build` or remove the PROD check temporarily to build the variant in development
+	...(PROD && BUILD_VARIANT
```

Comment:
What if the local user has forced themselves into a test? You’ll want to make sure that this case is covered otherwsie they will not get any JS

Reply:
I might be wrong so please verify but aiui...

When Fastly assigns a user to AB test buckets then I thought those cookies / headers only apply to the *.theguardian.com and not localhost? So aiui there is no chance of auto assignment locally?

In which case the only way a dev can be part of the variant build locally is if they manually force themselves into the test in which case they can uncomment the check locally too?

Reply:
I think that seems fair!

Reply:
Thanks for checking. If it is a problem for anyone it's easy to revert.

Reply:
When testing SWC locally I was just switching the variant build in webpack to be the modern :)